reactor-netty-http/src/test/java/reactor/netty/http/server/HttpServerTests.java

```diff
@@ -975,14 +975,16 @@ void httpServerRequestConfigInjectAttributes() {
 		AtomicBoolean validate = new AtomicBoolean();
 		AtomicInteger chunkSize = new AtomicInteger();
 		AtomicBoolean allowDuplicateContentLengths = new AtomicBoolean();
+		AtomicBoolean allowPartialChunks = new AtomicBoolean();
```

Comment:
```suggestion
		AtomicBoolean allowPartialChunks = new AtomicBoolean(true);
```

Reply:
Then below you need to add the actual value

```
allowPartialChunks.set((Boolean) getValueReflection(decoder, "allowPartialChunks", 2));
```

Reply:
Oh, somehow missed it for server tests, thank you